push_notifications/conf/app.py

```diff
@@ -0,0 +1,357 @@
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.six import string_types
+from .base import BaseConfig, check_apns_certificate
+from ..settings import PUSH_NOTIFICATIONS_SETTINGS as SETTINGS
+
+"""
```

Comment:
Could you move this massive docstring somewhere it's more likely to be seen by a user?

We don't have doc pages right now, so either:

- The wiki
- The README.rst (though it'd have to be more terse)
- An example settings file you ship alongside the rest of the code.